network/api.go

```diff
@@ -35,6 +36,11 @@ type Driver interface {
 	Leave(*LeaveRequest) error
 }
 
+// GetCapabilitiesResponse returns whether or not this network is global or local
```

Comment:
It's about local/global scope for now, but I'd try and make this comment more generic as that could change in future.

E.g 
`GetCapabilitiesResponse returns the drivers capabilities`